app/models/opensearch.rb

```diff
@@ -88,19 +88,19 @@ def matches
   # https://www.elastic.co/guide/en/elasticsearch/reference/current/query-filter-context.html
   def filters
     f = []
-    f.push filter(@params[:collection], 'collections') if @params[:collection]
-    f.push filter(@params[:contributor], 'contributors') if @params[:contributor]
+    f.push filter(@params[:collection_facet], 'collections') if @params[:collection_facet]
```

Comment:
Is the `collection_facet` filter supposed to be addressed in the `query_type` file as well? I see the other seven fields, but not this one.

Reply:
Yeah I think so. It's complicated because some of the facets don't actually work at all right now... but I think you are correct in that it won't show up as an option in GraphQL until the `query_type` file is updated.